From 8417a6c0a5d41085029bdf71d62348f20422e110 Mon Sep 17 00:00:00 2001
From: tromey <tromey@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Mon, 11 Feb 2002 00:01:11 +0000
Subject: [PATCH] 2002-02-10  Rainer Orth  <ro@TechFak.Uni-Bielefeld.DE>

	* lib/libjava.exp (libjava_find_lib): Remove duplicate .la files.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/trunk@49659 138bc75d-0d04-0410-961f-82ee72b054a4
---
 libjava/testsuite/ChangeLog       |    4 ++++
 libjava/testsuite/lib/libjava.exp |   10 +++++++---
 2 files changed, 11 insertions(+), 3 deletions(-)

diff --git a/libjava/testsuite/lib/libjava.exp b/libjava/testsuite/lib/libjava.exp
index 9b45f3c..67f9b6e 100644
--- a/libjava/testsuite/lib/libjava.exp
+++ b/libjava/testsuite/lib/libjava.exp
@@ -1,4 +1,4 @@
-# Copyright (C) 1998, 1999, 2000, 2001 Free Software Foundation
+# Copyright (C) 1998, 1999, 2000, 2001, 2002 Free Software Foundation
 
 load_lib "libgloss.exp"
 
@@ -175,11 +175,15 @@ proc libjava_find_lib {dir name} {
     foreach sub {.libs _libs} {
 	if {$gp != ""} {
 	    if {[file exists $gp/$dir/$sub/lib${name}.a]} then {
-		return "$gp/$dir/lib${name}.la -L$gp/$dir/$sub"
+		# Just return the `-L' option.  The library itself
+		# will be picked up via the spec file.
+		return "-L$gp/$dir/$sub"
 	    }
 	}
+	# Just return the `-L' option.  The library itself will be
+	# picked up via the spec file.
 	set lib [findfile $base_dir/../../$dir/$sub/lib${name}.a \
-		   "$base_dir/../../$dir/lib${name}.la -L$base_dir/../../$dir/$sub" \
+		   "-L$base_dir/../../$dir/$sub" \
 		   ""]
 	if {$lib != ""} {
 	    return $lib
-- 
1.7.8.2

